diff --git a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
index 766e840..fff553e 100644
--- a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
+++ b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
@@ -531,6 +531,7 @@ abstract class Kernel implements KernelInterface, TerminableInterface
         $fresh = true;
         if (!$cache->isFresh()) {
             $container = $this->buildContainer();
+            var_dump($container->getDefinitions());die;
             $container->compile();
             $this->dumpContainer($cache, $container, $class, $this->getContainerBaseClass());
 
