diff --git a/dino_container/roar.php b/dino_container/roar.php
index d07d0ad..036eb42 100644
--- a/dino_container/roar.php
+++ b/dino_container/roar.php
@@ -3,8 +3,9 @@
 namespace Dino\Play;
 
 use Monolog\Handler\StreamHandler;
-use Monolog\Logger;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
 
 require __DIR__.'/../vendor/autoload.php';
 
@@ -12,8 +13,14 @@ $container = new ContainerBuilder();
 
 $handler = new StreamHandler(__DIR__.'/dino.log');
 $container->set('logger.stream_handler', $handler);
-$logger = new Logger('main', array($container->get('logger.stream_handler')));
-$container->set('logger', $logger);
+
+$loggerDefinition = new Definition('Monolog\Logger');
+$loggerDefinition->setArguments(array(
+    'main',
+    array(new Reference('logger.stream_handler'))
+));
+$container->setDefinition('logger', $loggerDefinition);
+
 runApp($container);
 
 function runApp(ContainerBuilder $container)
