diff --git a/app/config/services.yml b/app/config/services.yml
index 5c76fc5..649b432 100644
--- a/app/config/services.yml
+++ b/app/config/services.yml
@@ -4,6 +4,7 @@ parameters:
 #    parameter_name: value
 
 services:
-#    service_name:
-#        class: AppBundle\Directory\ClassName
-#        arguments: ["@another_service_name", "plain_value", "%parameter_name%"]
+    user_agent_subscriber:
+        class: AppBundle\EventListener\UserAgentSubscriber
+        tags:
+            - { name: kernel.event_subscriber }
diff --git a/src/AppBundle/EventListener/UserAgentSubscriber.php b/src/AppBundle/EventListener/UserAgentSubscriber.php
new file mode 100644
index 0000000..e1fea4d
--- /dev/null
+++ b/src/AppBundle/EventListener/UserAgentSubscriber.php
@@ -0,0 +1,22 @@
+<?php
+
+namespace AppBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class UserAgentSubscriber implements EventSubscriberInterface
+{
+    public function onKernelRequest()
+    {
+        die('does it work?');
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return array(
+            // equal to kernel.request
+            KernelEvents::REQUEST => 'onKernelRequest'
+        );
+    }
+}
