diff --git a/src/AppBundle/EventListener/UserAgentSubscriber.php b/src/AppBundle/EventListener/UserAgentSubscriber.php
index 6c85523..c002379 100644
--- a/src/AppBundle/EventListener/UserAgentSubscriber.php
+++ b/src/AppBundle/EventListener/UserAgentSubscriber.php
@@ -2,6 +2,19 @@
 
 namespace AppBundle\EventListener;
 
-class UserAgentSubscriber
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class UserAgentSubscriber implements EventSubscriberInterface
 {
+    public function onKernelRequest()
+    {
+        die('it works');
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return array(
+            'kernel.request' => 'onKernelRequest'
+        );
+    }
 }
