diff --git a/src/AppBundle/EventListener/UserAgentSubscriber.php b/src/AppBundle/EventListener/UserAgentSubscriber.php
index ec3bcd6..729f374 100644
--- a/src/AppBundle/EventListener/UserAgentSubscriber.php
+++ b/src/AppBundle/EventListener/UserAgentSubscriber.php
@@ -14,8 +14,9 @@ class UserAgentSubscriber implements EventSubscriberInterface
         $this->logger = $logger;
     }
 
-    public function onKernelRequest()
+    public function onKernelRequest($event)
     {
+        dump($event);
         $this->logger->info('Yea, it totally works!');
     }
 
