diff --git a/app/cache/dev/appDevDebugProjectContainer.php b/app/cache/dev/appDevDebugProjectContainer.php
index 14764a6..316aa98 100644
--- a/app/cache/dev/appDevDebugProjectContainer.php
+++ b/app/cache/dev/appDevDebugProjectContainer.php
@@ -3827,7 +3827,7 @@ class appDevDebugProjectContainer extends Container
      */
     protected function getUserAgentSubscriberService()
     {
-        return $this->services['user_agent_subscriber'] = new \AppBundle\EventListener\UserAgentSubscriber();
+        return $this->services['user_agent_subscriber'] = new \AppBundle\EventListener\UserAgentSubscriber($this->get('logger'));
     }
 
     /**
diff --git a/app/config/services.yml b/app/config/services.yml
index 649b432..3a3aaf6 100644
--- a/app/config/services.yml
+++ b/app/config/services.yml
@@ -6,5 +6,6 @@ parameters:
 services:
     user_agent_subscriber:
         class: AppBundle\EventListener\UserAgentSubscriber
+        arguments: ["@logger"]
         tags:
             - { name: kernel.event_subscriber }
diff --git a/src/AppBundle/EventListener/UserAgentSubscriber.php b/src/AppBundle/EventListener/UserAgentSubscriber.php
index c002379..ec3bcd6 100644
--- a/src/AppBundle/EventListener/UserAgentSubscriber.php
+++ b/src/AppBundle/EventListener/UserAgentSubscriber.php
@@ -2,13 +2,21 @@
 
 namespace AppBundle\EventListener;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
 class UserAgentSubscriber implements EventSubscriberInterface
 {
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
     public function onKernelRequest()
     {
-        die('it works');
+        $this->logger->info('Yea, it totally works!');
     }
 
     public static function getSubscribedEvents()
