diff --git a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
index 02ec357..e755f4b 100644
--- a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
+++ b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
@@ -645,7 +645,7 @@ abstract class Kernel implements KernelInterface, TerminableInterface
         if (null !== $cont = $this->registerContainerConfiguration($this->getContainerLoader($container))) {
             $container->merge($cont);
         }
-        var_dump($container->getDefinitions());die;
+        var_dump($container->getParameterBag()->all());die;
 
         $container->addCompilerPass(new AddClassesToCachePass($this));
 
