diff --git a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
index 9d867cd..d3eed40 100644
--- a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
+++ b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
@@ -639,7 +639,6 @@ abstract class Kernel implements KernelInterface, TerminableInterface
         }
 
         $container = $this->getContainerBuilder();
-        var_dump($container);die;
         $container->addObjectResource($this);
         $this->prepareContainer($container);
 
