diff --git a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
index 26e15d1..766e840 100644
--- a/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
+++ b/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php
@@ -614,7 +614,6 @@ abstract class Kernel implements KernelInterface, TerminableInterface
         }
 
         $container = $this->getContainerBuilder();
-        var_dump($container);die;
         $container->addObjectResource($this);
         $this->prepareContainer($container);
 
