diff --git a/src/AppBundle/EventListener/UserAgentSubscriber.php b/src/AppBundle/EventListener/UserAgentSubscriber.php
index ab7d82f..c784715 100644
--- a/src/AppBundle/EventListener/UserAgentSubscriber.php
+++ b/src/AppBundle/EventListener/UserAgentSubscriber.php
@@ -28,6 +28,10 @@ class UserAgentSubscriber implements EventSubscriberInterface
             $response = new Response('Come back later');
             //$event->setResponse($response);
         }
+
+        $request->attributes->set('_controller', function($id) {
+            return new Response('Hello '.$id);
+        });
     }
 
     public static function getSubscribedEvents()
