diff --git a/src/AppBundle/Controller/DinosaurController.php b/src/AppBundle/Controller/DinosaurController.php
index ddb793c..75fee6c 100644
--- a/src/AppBundle/Controller/DinosaurController.php
+++ b/src/AppBundle/Controller/DinosaurController.php
@@ -5,6 +5,8 @@ namespace AppBundle\Controller;
 use AppBundle\Entity\Dinosaur;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
 
 class DinosaurController extends Controller
 {
@@ -17,6 +19,17 @@ class DinosaurController extends Controller
             ->getRepository('AppBundle:Dinosaur')
             ->findAll();
 
+        $request = new Request();
+        $request->attributes->set(
+            '_controller',
+            'AppBundle:Dinosaur:_latestTweets'
+        );
+        $httpKernel = $this->container->get('http_kernel');
+        $response = $httpKernel->handle(
+            $request,
+            HttpKernelInterface::SUB_REQUEST
+        );
+
         return $this->render('dinosaurs/index.html.twig', [
             'dinos' => $dinos,
             'isMac' => $isMac
