diff --git a/app/Resources/views/dinosaurs/_latestTweets.html.twig b/app/Resources/views/dinosaurs/_latestTweets.html.twig
index 636858c..c108b9f 100644
--- a/app/Resources/views/dinosaurs/_latestTweets.html.twig
+++ b/app/Resources/views/dinosaurs/_latestTweets.html.twig
@@ -1,9 +1,11 @@
 
 <div class="navbar-left tweets">
     <p class="text-center">Tweets from T-Rex Problems</p>
+    <p class="text-center">{{ isMac ? 'on a Mac' : 'Not on a Mac' }}</p>
     <ul>
         {% for tweet in tweets %}
             <li>{{ tweet }}</li>
         {% endfor %}
     </ul>
 </div>
+
diff --git a/src/AppBundle/Controller/DinosaurController.php b/src/AppBundle/Controller/DinosaurController.php
index 92fd9cd..466ea54 100644
--- a/src/AppBundle/Controller/DinosaurController.php
+++ b/src/AppBundle/Controller/DinosaurController.php
@@ -35,7 +35,7 @@ class DinosaurController extends Controller
         ]);
     }
 
-    public function _latestTweetsAction()
+    public function _latestTweetsAction($isMac)
     {
         $tweets = [
             'Dinosaurs can have existential crises too you know.',
@@ -44,7 +44,8 @@ class DinosaurController extends Controller
         ];
 
         return $this->render('dinosaurs/_latestTweets.html.twig', [
-            'tweets' => $tweets
+            'tweets' => $tweets,
+            'isMac' => $isMac
         ]);
     }
-} 
\ No newline at end of file
+}
\ No newline at end of file
diff --git a/src/AppBundle/EventListener/UserAgentSubscriber.php b/src/AppBundle/EventListener/UserAgentSubscriber.php
index 23fd225..87117f6 100644
--- a/src/AppBundle/EventListener/UserAgentSubscriber.php
+++ b/src/AppBundle/EventListener/UserAgentSubscriber.php
@@ -24,6 +24,9 @@ class UserAgentSubscriber implements EventSubscriberInterface
         $this->logger->info('Hello there browser: '.$userAgent);
 
         $isMac = stripos($userAgent, 'Mac') !== false;
+        if ($request->query->get('notMac')) {
+            $isMac = false;
+        }
         $request->attributes->set('isMac', $isMac);
     }
 
